fragments/master-ansible.sh

```diff
@@ -238,15 +242,34 @@ EOF
 EOF
     fi
 
-    ansible-playbook --inventory /var/lib/ansible/inventory /var/lib/ansible/services.yml
+    ansible-playbook \
+        --inventory /var/lib/ansible/inventory \
+        /var/lib/ansible/services.yml \
+        2>&1 > /var/log/ansible-services-$$.log
```

Comment:
this is not a valid redirect - stderr is redirected to "old" stdout (/dev/stdout instead of /var/log/ansible-services-$$.log) - you need to swap it as it is in the first change